server_admin/topics/users/proc-configuring-user-attributes.adoc

```diff
@@ -11,7 +11,7 @@ User attributes provide a customized experience for each user. You can create a
 * You are in the realm where the user exists.
 
 .Procedure
-. Click *Users* in the left menu. 
+. Click *Users* in the main menu. 
 . Select a user to manage.
 . Click the *Attributes* tab.
 . Enter the attribute name in the *Key* field.
```

Comment:
I found the original screen helpful to understand this procedure.  Does it create a problem to keep it?